bin/docker-entrypoint

```diff
@@ -41,6 +51,7 @@ help() {
   echo "server -- start Redash server (with gunicorn)"
   echo "worker -- start Celery worker"
   echo "scheduler -- start Celery worker with a beat (scheduler) process"
+  echo "dev_scheduler -- start Celery worker with a beat (scheduler) process which picks up code changes and reloads"
```

Comment:
Let's name this `dev_worker` as it better describes what it does.

Reply:
f5f707e